modules/reference/pages/cluster-properties.adoc

```diff
@@ -935,17 +935,21 @@ Each throughput control group consists of:
 
 Example values:
 
-* `[{'name': 'first_group','client_id': 'client1'}, {'name': 'catch all'}]`
-* `[{'client_id': 'consumer-\d+'}]`
+* `[{'name': 'first_group','client_id': 'client1'}, {'client_id': 'consumer-\d+'}]`
+* `[{'name': 'catch all'}]`
 
-A connection is assigned the first matching group and is then excluded from throughput control.
+A connection is assigned the first matching group and is then excluded from throughput control. You can optionally omit the `client_id` and specify only a `name`, as shown. In this situation, all clients will match the rule and Redpanda will exclude them from all from node-wide throughput control.
```

Comment:
Can you check your wording here, please.

You can optionally omit `client_id` independently of `name`

You may specify `{"client_id": "+empty"}` which matches when a client does not send a `client_id`.